### PR TITLE
Completed Game

### PR DESCRIPTION
**Pull request by barakisbrown**

### CODE REVIEW

File: MathGame.barakisbrown/MathGame.barakisbrown/Problems/Division.cs [hunk 1]
```diff
@@ -0,0 +1,55 @@
+﻿namespace MathGame.barakisbrown.Problems;
+
+internal class Division : IProblem
```
Comment (TheCSharpAcademy) [inline MathGame.barakisbrown/MathGame.barakisbrown/Problems/Division.cs:3]: The division game should only have questions that result in integers, as per the 2nd requirement. Let me know if you need help solving it. 

image

**Review by TheCSharpAcademy — CHANGES_REQUESTED** [review]
Hi there! The project looks great. The game does everything it's supposed to do and I like how you used inheritance 😁 It's also very clean and tidy. 

Could you please just fix the division issue so I can approve it?

File: MathGame.barakisbrown/MathGame.barakisbrown/Problems/Division.cs (hunk 1 above)
Comment (barakisbrown) [reply MathGame.barakisbrown/MathGame.barakisbrown/Problems/Division.cs:3]: Division issue has been resolved.

**Comment by TheCSharpAcademy** [comment]
@barakisbrown Hi there! Just checking if you need any help committing your updates? 😁

**Comment by barakisbrown** [comment]
I did do the commits that you asked about.  I told you in discord that I have closed it saying it was completed.

**Comment by barakisbrown** [comment]
DOH .. I forgot to push what I had done.  Now it is finished

**Review by TheCSharpAcademy — CHANGES_REQUESTED** [review]
Hey! The issue persists. You can't allow your program to show to the users equations that result in non-integers. Let me know you if you need help 🤓

**Comment by barakisbrown** [comment]
I have fixed the issue by looping until the two numbers give me an integer.

**Pull request merged**